test-helpers/src/docker_compose.rs

```diff
@@ -48,6 +48,18 @@ pub static IMAGE_WAITERS: [Image; 11] = [
         log_regex_to_wait_for: r"Cluster state changed|Cluster correctly created",
```

Comment:
we should be able to delete `library/redis:6.2.5`, `bitnami/redis:6.2.13-debian-11-r73` and `bitnami/redis-cluster:6.2.12-debian-11-r26` now

Reply:
I can see `redis-cluster:6.2.12-debian-11-r26` referenced in `shotover-proxy/benches/windsock/redis/bench.rs`. Should I still delete?

Reply:
I think the other 2 can be deleted as they are not referenced elsewhere.

Reply:
good point!
We'll need to keep `redis-cluster:6.2.12-debian-11-r26` in IMAGE_WAITERS until we change windsock benches.